packages/react-events/src/__tests__/Press-test.internal.js

```diff
@@ -2347,13 +2345,32 @@ describe('Event responder: Press', () => {
     root.render(<MyComponent />);
     Scheduler.flushAll();
 
-    dispatchEvent('pointerdown');
-    dispatchEvent('pointerup');
-    dispatchEvent('click');
+    dispatchEvent('pointerdown', 100);
+    dispatchEvent('pointerup', 100);
+    dispatchEvent('click', 100);
 
-    expect(renderCounts).toBe(2);
+    if (__DEV__) {
+      expect(renderCounts).toBe(2);
+    } else {
+      expect(renderCounts).toBe(1);
+    }
     Scheduler.flushAll();
-    expect(renderCounts).toBe(4);
+    if (__DEV__) {
+      expect(renderCounts).toBe(4);
+    } else {
+      expect(renderCounts).toBe(2);
+    }
+
+    dispatchEvent('pointerdown', 100);
+    dispatchEvent('pointerup', 100);
+    // Ensure the timeStamp logic works
```

Comment:
Is the Press event component the best place to test this logic?

Reply:
It’s fine for now. All responders should have some variant of this too.

Reply:
It's that the changes are being made to the core but the tests are here. And those tests have `__DEV__`-specific results

Reply:
We can follow up after and make tests in core too. I don't want to block this any longer given I'm on PTO in a few days. Maybe we can add this to our TODO list otherwise.